tox.ini

```diff
@@ -3,19 +3,21 @@ linting_folders=src/wagtailbakery/ tests/ examples/
 
 [tox]
 envlist=
-  py{38,39,310,311}-django{32,40,41}-wagtail{41,42} # Wagtail 4.1 LTS and 4.2 - all supported Python - all Django
-  py{38,39,310,311}-django{32,41,42}-wagtail50      # Wagtail 5.0 - all supported Python - all Django
-  wagtailmain                                       # Wagtail main latest compatible version
+  py{38,39,310,311}-django{32,40,41}-wagtail41          # Wagtail 4.1 LTS - all supported Python - all Django
+  py{38,39,310,311,312}-django{32,41,42}-wagtail{50,51,52}  # Wagtail 5.0, 5.1 and 5.2 LTS - all supported Python - all Django
+  wagtailmain                                           # Wagtail main latest compatible version
```

Comment:
As per our README, we aim to officially support LTS + current, so today it would mean 5.2 only. I guess we decided on this at some point to lessen the maintenance need, but there's also no reason not to support previous versions if there is no code change or compatibility layer to make it work. So I think we can merge like this.